### DISCUSSION TITLE
Is there any reason to include description and type in history.db?

### DISCUSSION THREAD

**Discussion by Querulous** [FreeTubeApp/FreeTube#4698] [General]
.description depending on video can be very long, will quickly bloat .db file especially when it contains duplicated entries. I haven't noticed any place where this info is used in app. I thought it might be used in "Search in history", but it's looking only for video title.

.type in my .db (~10K entries) it always has value "video"

**Comment by absidue** [comment]
The description is used on the history page if you set the layout to `list` (you will also see descriptions on the search page for example).

As for the `type` that is used by the app to identify what type of list item it is, as our list/grid view can handle videos, channels, playlists and posts.
